src/server/pps/server/api_server.go

```diff
@@ -79,6 +79,10 @@ const (
 	// default we return logs from up to 24 hours ago.
 	DefaultLogsFrom  = time.Hour * 24
 	ppsTaskNamespace = "/pps"
+
+	// kubernetesNameLimit is the maximum length of a ReplicationController
+	// or Service name.
+	kubernetesNameLimit = 63
```

Comment:
This is a slightly inaccurate name.  It's really services that are to blame, which then blames some ancient RFCs: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names

RFC1123, incidentally, says that clients MUST understand 63 character DNS names, but SHOULD understand 255.  RFC1035 mandates 63 without explanation, and mandates 255 for entire DNS paths with the explanation that it's "for simplicity".  RFC1035 is from 1987 😂

Reply:
dnsNameLimit?